salt/modules/pacman.py

```diff
@@ -236,6 +236,158 @@ def list_pkgs(versions_as_list=False, **kwargs):
     return ret
 
 
+def group_list():
+    '''
+    .. versionadded:: ?
```

Comment:
All of the versionadded/versionchanged directives here should have ``?`` replaced with ``Carbon``, the codename for the next feature release.